.gitignore

```diff
@@ -6,8 +6,6 @@ _build
 dist/
 *.pyc
 .pytest_cache/
-_version.py
-version.txt
```

Comment:
So these files are untracked but expected to be included in the distribution? How do we prevent committing them?

Reply:
This repo used to use setuptools-scm.  Now that we use hatch, we want the version file to be there.

Reply:
Shouldn't this PR be adding the file in that case?

Reply:
The file was already there, hatch uses `.gitignore` to determine what goes into the dist by default, unless you override it.

Reply:
Oh I see the file already exists. I forgot you could commit a .gitignore file.